_projects/dnn_cfd.md

```diff
@@ -13,47 +13,29 @@ topics:
 keywords:
   - CFD
   - deep neural networks
-  - autoencoders
-  - generative adversial networks
-  - transfer learning
+  - physics-informed neural networks
+  - reduced order models
+  - graph neural networks
 head: lintermann_a
 members:
   - tsubokura_m
-  - onishi_k
+  - onishi_y
```

Comment:
Are you sure about the change in the first name here? If so, you need to edit this in the database, too.

Reply:
thanks, I have updated the database accordingly